### PR TITLE
Rename server-slave to status-module-client

### PR DESCRIPTION
**Pull request by RickvdStaaij**
### What

@steefmin made me aware that `server-client` is a confusing name, since its more of a client than a server, which is totally correct.

Not sure yet if this is the best fit.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -74,8 +74,8 @@ do-show-commands:
 	@echo "    make preview-docs               Run a live preview of the documentation."
 	@echo "\nDevelopment:"
 	@echo "    make dev-server                 Run the development server."
-	@echo "    make dev-server-slave           Run the development slave server, listening to a master."
-	@echo "    make dev-client                 Build, run and watch the development dashboard."
+	@echo "    make dev-module-client          Run the development slave server, listening to a master."
```
Comment (eXistenZNL) [inline Makefile]: Please update the description of this make command.

File: README.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ To start development run:
 
 -   `make init`: The first time you want to run the project locally
 -   `make dev-server`: Run the development server
--   `make dev-server-slave`: Run the development server slave that listens to a development server
+-   `make dev-module-client`: Run the development server slave that listens to a development server
```
Comment (eXistenZNL) [inline README.md]: Do you even update the documentation bro?

File: back-end/domain/socket/ConnectionManager.js [hunk 3]
```diff
@@ -89,7 +89,7 @@ class SocketConnectionManager {
     pushVideo(videoDetails) {
         if (!this.io) {
             // No socket connection is established, no need to emit anything
-            console.log(`[SocketConnectionManager] No socket connection. Videos can't be pushed on server-slaves.`);
+            console.log(`[SocketConnectionManager] No socket connection. Videos can't be pushed on module-clients.`);
```
Comment (eXistenZNL) [inline back-end/domain/socket/ConnectionManager.js]: Apparently you do sometimes.

File: dev/docker/module-client/.dockerignore [hunk 4]
```diff
@@ -2,7 +2,8 @@
 **
 
 # Allow files and directories
+!/back-end
```
Comment (eXistenZNL) [inline dev/docker/module-client/.dockerignore]: DO

File: dev/docker/module-client/.dockerignore [hunk 5]
```diff
@@ -2,7 +2,8 @@
...
+!/config
```
Comment (eXistenZNL) [inline dev/docker/module-client/.dockerignore]: YOU

File: dev/docker/module-client/.dockerignore [hunk 6]
```diff
@@ -2,7 +2,8 @@
...
 !/node_modules
```
Comment (eXistenZNL) [inline dev/docker/module-client/.dockerignore]: EVEN

File: dev/docker/module-client/.dockerignore [hunk 7]
```diff
@@ -2,7 +2,8 @@
...
-!/shared
```
Comment (eXistenZNL) [inline dev/docker/module-client/.dockerignore]: LIFT

File: dev/docker/module-client/.dockerignore [hunk 8]
```diff
@@ -2,7 +2,8 @@
...
-!/server
```
Comment (eXistenZNL) [inline dev/docker/module-client/.dockerignore]: BRO

File: docs/setup/configuration.md [hunk 9]
```diff
@@ -36,7 +36,7 @@ The `server` object must be configured as following:
 
 ## Server slave configuration
 
-The `serverSlave` configuration is required when you're running the `server/server-slave.js` application!
+The `serverSlave` configuration is required when you're running the `back-end/module-client.js` application!
```
Comment (eXistenZNL) [inline docs/setup/configuration.md]: Do you even LIFT BRO

File: Makefile (hunk 1 above)
Comment (RickvdStaaij) [reply Makefile]: Fixed.

File: README.md (hunk 2 above)
Comment (RickvdStaaij) [reply README.md]: Fixed.

File: dev/docker/module-client/.dockerignore (hunk 5 above)
Comment (RickvdStaaij) [reply dev/docker/module-client/.dockerignore]: Deze is lekker

File: dev/docker/module-client/.dockerignore (hunk 6 above)
Comment (RickvdStaaij) [reply dev/docker/module-client/.dockerignore]: NEED

File: docs/setup/configuration.md (hunk 9 above)
Comment (RickvdStaaij) [reply docs/setup/configuration.md]: Replaced `serverSlave` config key with `moduleClient`

**Pull request merged**